pkg/application/options.go

```diff
@@ -288,18 +249,18 @@ func WithProducerDaemon(app *App) {
 }
 
 func WithTracing(app *App) {
-	app.addLoggerOption(func(config cfg.GosoConf, logger mon.GosoLog) error {
-		tracingHook := tracing.NewLoggerErrorHook()
+	app.addLoggerOption(func(config cfg.GosoConf, logger log.GosoLogger) error {
+		tracingHook := tracing.NewLoggerErrorHandler()
```

Comment:
Should we rename this from hook to handler in the variable?

Reply:
renamed